tests/spec/fs.unlink.spec.js

```diff
@@ -104,3 +104,30 @@ describe('fs.unlink', function() {
     });
   });
 });
+
+describe('fsPromises.stat', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should not unlink directories (using promises)', () => {
+    var fsPromises = util.fs().promises;
+  
+    return fsPromises.mkdir('/mydir')
+      .then(() => {
+        fsPromises.unlink('/mydir')
+          .catch((error) => {
+            expect(error).to.exist;
+            expect(error.code).to.equal('EPERM');
+            fsPromises.stat('/mydir')
+              .then((error, stats) =>{
+                expect(error).not.to.exist;
+                expect(stats).to.exist;
+                expect(stats.type).to.equal('DIRECTORY');
+              });
+          });
+      })
+      .catch((error) => {
+        if (error) throw error;
+      });
+  });
+}); 
```

Comment:
Hey @mphan6 - nice job here 👏 ! Just one _tiny_ nitpick - GitHub is telling us that you didn't add a blank line at the end of the file with the  ![image](https://user-images.githubusercontent.com/13009507/45987423-5b2dc900-c03f-11e8-951f-99264009b35a.png) icon.

Can you amend that to your existing commit then force push again? Let me know if you need any help with that!